tests/components/zha/test_switch.py

```diff
@@ -461,14 +462,18 @@ async def test_switch_configurable(
     cluster.write_attributes.reset_mock()
     cluster.write_attributes.side_effect = ZigbeeException
 
-    await hass.services.async_call(
-        SWITCH_DOMAIN, "turn_off", {"entity_id": entity_id}, blocking=True
-    )
+    with pytest.raises(HomeAssistantError):
+        await hass.services.async_call(
+            SWITCH_DOMAIN, "turn_off", {"entity_id": entity_id}, blocking=True
+        )
 
-    assert len(cluster.write_attributes.mock_calls) == 1
```

Comment:
Is that a different behavior in commands or just a different test checks (len(calls) =1 vs 3)

Reply:
Retrying is enabled for many more types of commands (some code was directly accessing the zigpy cluster instead of going through the ZHA proxy) so the number of sent requests will be higher in failure cases.

Reply:
Retrying is now enabled for many more types of commands (some code was directly accessing the zigpy cluster instead of going through the ZHA proxy) so the number of sent requests will be higher in failure cases.